ui/components/FilterBar.tsx

```diff
@@ -0,0 +1,157 @@
+import {
+  Checkbox,
+  Chip,
+  Divider,
+  List,
+  ListItem,
+  ListItemIcon,
+  Popover,
+} from "@material-ui/core";
+import * as React from "react";
+import styled from "styled-components";
+import { theme } from "..";
+import Button from "./Button";
+import Flex from "./Flex";
+import Icon, { IconType } from "./Icon";
+import Input from "./Input";
+import Spacer from "./Spacer";
+import Text from "./Text";
+
+/** Filter Bar Properties */
+export interface Props {
+  className?: string;
+  /** currently checked filter options mapped to chips next to filter icon. Part of a `useState` with `setActiveFilters` */
+  activeFilters: string[];
+  /** the setState function for `activeFilters` */
+  setActiveFilters: (newFilters: string[]) => void;
+  /** Object containing column headers + corresponding filter options */
+  filterList: Record<string, string[]>;
+}
+
+const FilterHeader = styled(Flex)`
+  background: ${theme.colors.primary10};
+  color: white;
+  padding: ${theme.spacing.base};
+`;
+
+/** Form Filter Bar */
+function UnstyledFilterBar({
+  className,
+  activeFilters,
+  setActiveFilters,
+  filterList,
+}: Props) {
+  const [anchorEl, setAnchorEl] = React.useState(null);
```

Comment:
This should probably be a `ref`: https://reactjs.org/docs/hooks-reference.html#useref

Reply:
It doesn't work as a ref I tried it like a billion different ways. In the example in MUI docs they actually do it in state as well: https://mui.com/components/popover/